files/es/learn/css/css_layout/grids/index.html

```diff
@@ -657,9 +657,9 @@ <h6 id="Un_método_de_compaginación_en_cuadrícula_con_CSS">Un método de compa
 
 <p>{{ EmbedLiveSample('Grid_8', '100%', 400) }}</p>
 
-<p>Si usas el <a href="/en-US/docs/Tools/Page_Inspector/How_to/Examine_grid_layouts">inspector de cuadrícula de Firefox</a> para ver una superposición de las líneas de cuadrícula sobre tu diseño, puedes observar cómo funciona nuestra cuadrícula de 12 columnas.</p>
+<p>Si usas el <a href="https://firefox-source-docs.mozilla.org/devtools-user/page_inspector/how_to/examine_grid_layouts/index.html">inspector de cuadrícula de Firefox</a> para ver una superposición de las líneas de cuadrícula sobre tu diseño, puedes observar cómo funciona nuestra cuadrícula de 12 columnas.</p>
```

Comment:
Hey, @Jonatandb how was the weekend?
I have a question, where did you get this URL from? is it from the English version? can you provide a link, please?

Reply:
Hi Juan 🙋🏻‍♂️ everything fine, thanks! I hope you too.

Those links were suggested by the bot.

Should I rollback them?

Reply:
I searched those lines in the [English version](https://github.com/mdn/content/blob/main/files/en-us/learn/css/css_layout/grids/index.md) but didn't find it, it seems like those lines don't exist anymore.

I see you want to contribute, I'm wondering if could you change the whole file from HTML to Markdown as the English version is. please? 

This file https://github.com/mdn/translated-content/blob/main/files/es/learn/css/css_layout/grids/index.html should be exactly the same including format (.md) to this https://github.com/mdn/content/blob/main/files/en-us/learn/css/css_layout/grids/index.md but in Spanish :+1:

Reply:
Sorry Juan, I'm doing my best trying to help by making all this PR's but that you asked me seems to be a lot of work and honestly I don't like and I don't have enough time either to make full page translations (even my english is pretty basic too).

Hope you find soon someone else that can help you to translate the whole page.

Meanwhile I ask you: Should I continue making this PR's with little corrections or it's not so much useful since someone soon will translate the whole page anyway?

Reply:
> Sorry Juan, I'm doing my best trying to help by making all this PR's but that you asked me seems to be a lot of work and honestly I don't like and I don't have enough time either to make full page translations (even my english is pretty basic too).

well, almost the whole page is translated and you can still use the current Spanish wording to finish the whole page, you only need to check all that match with the English version. if you don't have time don't worry I was just letting you know a suggestion.  

> Hope you find soon someone else that can help you to translate the whole page.

yeah! eventually, we will have it translated.

> Meanwhile I ask you: Should I continue making this PRs with little corrections or it's not so much useful since someone soon will translate the whole page anyway?

yes, all contributions are welcome and important, (yours of course has been helpfully because you has been fixing all those links had been broken)

Reply:
said that, let's keep this PR rolling forward, I'm going to approve it, and let's see what the @mdn/yari-content-es

Reply:
Sure! Btw, it's the second time you tag that account but it always returns 404 when I click it 🙄

Reply:
maybe the URL is private, it must ping the ES team, it should redirect you to this URL https://github.com/orgs/mdn/teams/yari-content-es :thinking:  

![image](https://user-images.githubusercontent.com/7331511/163888256-f0502ba0-6da9-443e-a58d-d10afc940508.png)

Reply:
Definitely it's private then:

![image](https://user-images.githubusercontent.com/8082172/163888454-b06e68f0-33b0-4130-b5c2-e91ac5018cb5.png)

Reply:
it should be private, don't worry about it, the important thing is that they noticed we are calling them here :dagger:

Reply:
Hola! @JuanVqz las páginas de `tools` fueron redireccionadas a ese portal, de ahí provienen.

@Jonatandb  Mucha gracias por tu tiempo